ch6/exercise6-2/addAll_test.go

```diff
@@ -0,0 +1,50 @@
+package intset
+
+import (
+	"testing"
+)
+
+func TestAddAll(t *testing.T) {
+	setOne := &IntSet{}
+	setTwo := &IntSet{}
+	setThree := &IntSet{}
+	setOne.Add(1)
+	setOne.Add(6)
+	setOne.AddAll(7, 8, 9)
+	setTwo.Add(8)
+	setTwo.AddAll(24, 54)
+	setThree.Add(1)
+	setThree.AddAll(2, 3)
+
+	var tests = []struct {
+		set  *IntSet
+		want []int
+	}{
+		{setOne, []int{1, 6, 7, 8, 9}},
+		{setTwo, []int{8, 24, 54}},
+		{setThree, []int{1, 2, 3}},
+	}
```

Comment:
With this amount of setup/logic, table testing becomes hard to read -- there are triply nested loops and indentation 5 levels deep.  I'd recommend splitting out an equality helper, to simplify the top level test logic:

```go
func equal(want []int, got *IntSet) bool {
    // your equality code
}
```

Reply:
This is completed, I also added a toSlice() function in the test files for easier comparison